File: eval/fixtures/legacy-directory-backed-aspnet-commerce/src/Legacy/Billing/BillingRefundService.cs
namespace Acme.Legacy.Billing;

public class BillingRefundService
{
  private readonly BillingRefundRepository _billingRefundRepository = new BillingRefundRepository();

  public RefundAuditResult RebuildRefundAudit(string refundId)
  {
    return _billingRefundRepository.WriteRefundAuditEntry(refundId);
  }
}

File: eval/fixtures/legacy-directory-backed-spring-commerce/src/main/java/com/acme/legacy/billing/BillingRefundService.java
package com.acme.legacy.billing;

public class BillingRefundService {
  private final BillingRefundRepository billingRefundRepository = new BillingRefundRepository();

  public RefundAuditResult rebuildRefundAudit(String refundId) {
    return billingRefundRepository.writeRefundAuditEntry(refundId);
  }
}
